src/engraving/dom/segment.cpp

```diff
@@ -1095,6 +1095,25 @@ void Segment::setXPosInSystemCoords(double x)
     mutldata()->setPosX(x - measure()->x());
 }
 
+bool Segment::isInsideTuplet() const
+{
+    if (!isChordRestType()) {
```

Comment:
Re. https://github.com/musescore/MuseScore/issues/27087#issuecomment-2722983723: I think this early return is the cause. That raises the question: how to handle time tick segments here? They don't have chords, so we can't get the tuplet, so then we need to look at the next or previous segment and hope that that one is a ChordRest segment... not super streamlined.

And actually, I doubt the criterium should be `isInsideTuplet` at all. Shouldn't it rather be `isTickMultipleOfPowerOfTwo`? (i.e. 1/4, 3/8, but not 1/3)

Reply:
Yep, correct. In fact I knew that the proper solution is to check for a power-of-two denominator, but as a last-of-the-last-minute solution it felt safer to just mimick the existing "Cannot paste onto tuplets" logic. In fact you know what, I'll do this right now and merge it in 4.5.1 so we have proper time to test it.